### PR TITLE
Add 'web design standards' language to play3

### PR DESCRIPTION
**Pull request by mollieru**
Modifies checklist item to include link to the new U.S. Web Design Standards site.

### CODE REVIEW

**Comment by konklone** [comment]
You my be aware of this already, but https://playbook.cio.gov/designstandards is a 404.

**Comment by cew821** [comment]
URL works now -- I believe the design standards site was just soft-launched! :fireworks: :us: :fireworks:

File: _plays/03.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Make it simple and intuitive
 Using a government service shouldn’t be stressful, confusing, or daunting. It’s our job to build services that are simple and intuitive enough that users succeed the first time, unaided.
 
 #### checklist
-1. Create or use an existing, simple, and flexible design style guide for the service
+1. Use a simple and flexible design style guide for the service. Use the <a href="https://playbook.cio.gov/designstandards">U.S. Web Design Standards</a>as a default.
```
Comment (alex) [inline _plays/03.md]: Need a space after the `</a>`

**Comment by lukefretwell** [comment]
The "Mockup of VA Appeals Modernization Screen" link on https://pages.18f.gov/designstandards/ directs to [this image](https://pages.18f.gov/designstandards/assets-styleguide/img/home/example_voteUSAgov.png), which I assume is a bug.

**Comment by cew821** [comment]
thanks Luke - can you report this here?
https://github.com/18F/web-design-standards

**Comment by lukefretwell** [comment]
Apologies @cew821. Done.

**Comment by cew821** [comment]
No problem at all! It is a bit confusing with the shared domain. Thank you
so much for taking a look.

**Pull request merged**